Scripts/build.swift

```diff
@@ -61,19 +68,23 @@ enum Platform: String, CaseIterable, CustomStringConvertible {
             return "OS=7.2,name=Apple Watch Series 6 - 44mm"
         case .watchOS_8:
             return "OS=8.5,name=Apple Watch Series 6 - 44mm"
+        case .watchOS_9:
+            return "OS=9.0,name=Apple Watch Series 7 (45mm)"
```

Comment:
To confirm this friendly name changed from `-` to `(` `)`?

Reply:
yup. I don't know why but Xcode 14 did change the formatting here.